src/Libraries/CoreNodes/Color.cs

```diff
@@ -29,7 +29,7 @@ public byte Red
         /// <summary>
         ///     Find the green component of a color, 0 to 255.
         /// </summary>
-        /// <returns name="green">int between 0 and 255 inclusive.</returns>
+        /// <returns name="int">Green value for RGB color model, int between 0 and 255 inclusive.</returns>
```

Comment:
not sure what has been agreed with ADSK but it feels a bit inconsitent that we change `str` to `string`, `bool` to `boolean` and so on, but we still use `int` to be consistent that should probs be `integer`. Even though it feels a bit overkill, but so does the others

Reply:
addressed in 
c96366a